### ISSUE TITLE
Unable to trade conjured water

### ISSUE THREAD

**Issue by tehho** [liyunfan1223/mod-playerbots#452]
**Is your feature request related to a problem? Please describe.**
Unable to trade conjured water since it got no SellPrice

**Describe the solution you'd like**
Add exclusion to !item->GetTemplate()->SellPrice guard to take conjured items in to consideration

**Describe alternatives you've considered**
All trade of item without sellprice as long as no soul bound(?)

**Additional context**
N/A

**Comment by Grimfeather** [comment]
I don't understand. I'm able to trade conjured water with a mage.

I whisper the mage:   /w mage cast conjure water
and when i request trade the water is in the list of whispers
while the trade screen is open I whisper the mage the link of the conjured water by shift clicking on it.
the mage then puts the water in the trade window.

where does it go wrong for you?

**Comment by tehho** [comment]
After when I press trade.
I'm able to do all of the above, just not completing the trade transaction

**Comment by qoreycube** [comment]
Can confirm. Bots respond back with "[Conjured Fresh Water] - this item is not for sale."

**Comment by Loopher** [comment]
Within the last week or so, I noticed that I am no longer able to trade with a Mage bot to get summoned mana pies.  He refuses to trade saying it is not for sale.  Also, I am not certain which update it was when this became broken, but I used to be able to trade with the bots and retrieve quest items they looted.  Now they ask for money for the items.  Is there a new setting in the config for this?  I have not been able to find it and this is the only post I found where someone reported similar behavior.
